app/pandas.ipynb

```diff
@@ -210,9 +195,7 @@
    "cell_type": "code",
    "execution_count": 17,
    "outputs": [],
-   "source": [
-    "# Write your code here"
-   ],
+   "source": "df[\"Average age\"] = df[\"Age\"].mean()",
```

Comment:
Adding 'Average age' as a column in the DataFrame is not appropriate since it should be a single value, not a column. Consider calculating it separately and using it where needed.